src/main/java/dev/jbang/dependencies/DependencyUtil.java

```diff
@@ -54,7 +54,10 @@
 	public static final Pattern gavPattern = Pattern.compile(
 			"^(?<groupid>[^:]*):(?<artifactid>[^:]*)(:(?<version>[^:@]*))?(:(?<classifier>[^@]*))?(@(?<type>.*))?$");
 
-	public ModularClassPath resolveDependencies(List<String> deps, List<MavenRepo> repos,
+	private DependencyUtil() {
```

Comment:
hmm - I actually kept it non-static as some of these could benefit from caching but I can see it being okey...

Reply:
Yeah, I thought that might be the case, but I couldn't find any actual usage, nor something that would indicate we were going to need it in the near future. This way it's at least obvious right away that there's (currently) no state. If that ever changes it's not too hard to change it back.